### ISSUE TITLE
Guardian Node Keeps restarting

### ISSUE THREAD

**Issue by khamilton1971** [thetatoken/guardian-mainnet-guide#80]
After launching my guardian node, it sync'd but at the bottom it keeps cycling through "health check failed", "stopping node", starting node", over and over.  Ideas???

**Comment by DavidGarry7654** [comment]
What version of GN are you running on ?

**Comment by khamilton1971** [comment]
2.2.10

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Comment by DavidGarry7654** [comment]
Okay. Have you got a telegram messenger so i may  direct you to their telegram customer support ?

They’ll be happy to help you resolve your issue

**Comment by khamilton1971** [comment]
Unfortunately I do not have telegram messenger

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Comment by khamilton1971** [comment]
I have emailed Theta support directly but it will probably take them a while

**Comment by DavidGarry7654** [comment]
You can download the telegram messenger  and set it up. Make sure you add a username for your telegram account else your message will be marked as spam. 

Let me know when you’ve done that. So I’ll just redirect you to their support via web link.

**Comment by DavidGarry7654** [comment]
@khamilton1971 Have you set up your telegram account?

**Comment by khamilton1971** [comment]
No, I have heard that there is a bunch of disinformation about Theta on telegram and do not want anything to happen with my theta

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Comment by DavidGarry7654** [comment]
Oh okay. My bad. Here’s their DISCORD channel they use now https://discord.gg/ZKUfYk2z. You can reach them on there.

**Comment by GitHinse** [comment]
Hu guys, thanks for your replies, I suspected the issue could be caused by the older hardware I had put to the task. Mainly, it was using a mechanical Hard Drive. I swapped for SSD and and installed a new Node and it seems to work fine. I'm on v2.2.10 
Sorry I didn't reply fast, I'm managing a newborn and took some time to get around Telegram!
